dependencies.properties

```diff
@@ -1,5 +1,5 @@
 version=6.1.0-SNAPSHOT
-amf=5.4.7
+amf=5.4.8-RC.1
 amf.custom-validator.js=1.6.0
```

Comment:
I think we need use `1.7.0` as `amf.custom-validator-scalajs=0.5.8-RC.0`

Reply:
You're right!